Makefile

```diff
@@ -14,13 +14,48 @@ unit-test:
 clean:
 	rm -rf cellxgene_schema_cli/build cellxgene_schema_cli/dist cellxgene_schema_cli/cellxgene_schema.egg-info
 
+# RELEASE HELPERS
+
 pydist: clean
 	cd cellxgene_schema_cli && python setup.py sdist bdist_wheel
 
-test-release: pydist
+# Set PART=[major, minor, patch] as param to make bump.
+# This will create a new release candidate. (i.e. 0.16.1 -> 0.16.2-rc.0 for a patch bump)
+bump-version:
+	bumpversion --config-file .bumpversion.cfg $(PART)
+
+# This command is used to increment the rc number in case an initial candidate is erronous.
+bump-release-candidate:
+	bumpversion --config-file .bumpversion.cfg prerelversion --allow-dirty
+
+# This command is used to remove the `rc` tag from the release candidate after it has been successfully tested in Test PyPI.
+bump-release:
+	bumpversion --config-file .bumpversion.cfg prerel --allow-dirty
+
+# Create new version to commit to main
+# Set PART=[major, minor, patch]
+create-release-candidate: bump-version
+	@echo "Version bumped part:$(PART) and client built. Ready to commit and push"
+
+# Bump the release candidate version if needed (i.e. the previous release candidate had errors).
+recreate-release-candidate: bump-release-candidate
+	@echo "Version bumped part:$(PART) and client built. Ready to commit and push"
+
+# Build dist and release the release candidate to Test PyPI
+release-candidate-to-test-pypi: pydist
+	pip install twine
+	python -m twine upload --repository testpypi cellxgene_schema_cli/dist/*
+	@echo "Release candidate dist built and uploaded to test.pypi.org"
+
+# Build final dist (gets rid of the rc tag) and release final candidate to TestPyPI
+release-final-to-test-pypi: bump-release pydist
 	pip install twine
 	python -m twine upload --repository testpypi cellxgene_schema_cli/dist/*
+	@echo "Final release dist built and uploaded to test.pypi.org"
+	@echo "Please test the release on Test PyPI"
 
-release: pydist
+release-final: pydist
```

Comment:
suggest `release-to-pypi`, for consistency with `test`, and to make it clear you're changing the outside world.